packages/lib/plugin.js

```diff
@@ -355,6 +355,19 @@ class BaseMasterPlugin {
 	 */
 	async onInstanceStatusChanged(instance, prev) { }
 
+	/**
+	 * Called when the value of a master config field changed.
+	 *
+	 * Invoked after the value of the config field given by `field` has
+	 * changed on the master.
+	 *
+	 * @param {module:lib/config.ConfigGroup} group -
+	 *     The group who's field got changed on.
```

Comment:
The group who's field got changed ~~on.~~?

Reply:
Yes, that's better.